dvc/remote/local.py

```diff
@@ -46,22 +36,16 @@ class RemoteLOCAL(RemoteBASE):
 
     def __init__(self, repo, config):
         super().__init__(repo, config)
-        self.protected = config.get(Config.SECTION_CACHE_PROTECTED, False)
+        self.protected = config.get("protected", False)
 
-        shared = config.get(Config.SECTION_CACHE_SHARED)
+        shared = config.get("shared")
         self._file_mode, self._dir_mode = self.SHARED_MODE_MAP[shared]
 
         if self.protected:
             # cache files are set to be read-only for everyone
             self._file_mode = stat.S_IREAD | stat.S_IRGRP | stat.S_IROTH
 
-        cache_dir = config.get(Config.SECTION_REMOTE_URL)
-
-        if cache_dir is not None and not os.path.isabs(cache_dir):
-            cwd = config[Config.PRIVATE_CWD]
```

Comment:
@Suor You might've lost `_cwd` logic, which was resolving relative paths based on config location. Or am I missing something here?

Reply:
I thought that we have test for this 🤔

Reply:
This is not needed anymore:

> relative paths are handled transparently upon load/save

Reply:
@Suor Could you please refer me to some part of the code?

Reply:
Nevermind, I see it now.